src/test/java/core/basesyntax/service/impl/ReaderImplTest.java

```diff
@@ -0,0 +1,41 @@
+package core.basesyntax.service.impl;
```

Comment:
i would also recommend you to add a test with null file path, for expecting an exception. you need to cover edge cases in your tests.